code/src/components/ThoughtsInput.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+
+import HeartImage from "../assets/heart.png";
+
+
+
+export const ThoughtsInput = () => {
+    const ThoughtsUrl= "https://happy-thoughts-technigo.herokuapp.com/thoughts";
```

Comment:
Url variable name should not be written with capital letters

Reply:
thanks, fixed!